addons/spreadsheet/static/src/o_spreadsheet/o_spreadsheet.js

```diff
@@ -43178,6 +43178,7 @@
     css /* scss */ `
   .o-border-selector {
     padding: 4px;
+    background-color: white;
```

Comment:
Hello :)
Thanks for the fixes
`o_spreadsheet.js` and `o_spreadsheet.xml` are automatically generated by the [o-spreadsheet](https://github.com/odoo/o-spreadsheet) lib build tools. Your changes would be overwritten the next time we update the lib (once every 1-2 weeks).
These changes need to be in o-spreadsheet repo.
Do you want to do it (we don't have a -dev repo, you can push on origin ;)) ?
Or do you prefer to let us do it ?

Reply:
Hello @LucasLefevre , thanks for the heads-up  👍 
Changes has been applied  here https://github.com/odoo/o-spreadsheet/pull/3051